Assets/Script/GameManager.cs

```diff
@@ -1,25 +1,25 @@
-﻿using System.Collections;
+using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
 using System;
 
 public class GameManager : MonoBehaviour
 {
-    public Text PlayTime;
+    [SerializeField]
+    private Text PlayTime;
+    
     private TimeSpan startTime = new TimeSpan(0, 10, 0);
-private TimeSpan remainingTime;
-
-    private void Start()
+    public TimeSpan remainingTime { get; set; }
+    
+    private void OnEnable()
     {
         InvokeRepeating("CountDown", 0, 1);
     }
-
-
-
     private void CountDown()
     {
-        remainingTime = startTime - TimeSpan.FromSeconds(1);
-        PlayTime.text = remainingTime.ToString(@"mm\:ss")
+        startTime -= TimeSpan.FromSeconds(1);
+        remainingTime = startTime;
```

Comment:
이걸 startTime에 계산하는게 아니라 remainingTime에 계산하게 #35 코드리뷰 할 떄 적어 드렸었던 거 같아요.
```suggestion
        remainingTime -= TimeSpan.FromSeconds(1);
```